core/commands/object.go

```diff
@@ -242,7 +242,7 @@ var objectStatCmd = &cmds.Command{
 
 		fpath := path.Path(req.Arguments()[0])
 
-		object, err := objectGet(n, fpath)
+		object, err := core.Resolve(req.Context().Context, n, fpath)
```

Comment:
please see my comment below (on gh..). I have no strong feelings on this, can change it back as well.